standalone/standalone.tpl

```diff
@@ -1148,7 +1148,7 @@ def create_data_job(
 {{exec_data_processing_op_command}}
 """
     exec_data_processing_op_args = f"""
-data_processing_op(max_seq_len={MAX_SEQ_LEN}, max_batch_len={MAX_BATCH_LEN}, sdg="{DATA_PVC_SDG_PATH}", model="{DATA_PVC_MODEL_PATH}", skills_processed_data="{PREPROCESSED_DATA_SKILLS_PATH}", knowledge_processed_data="{PREPROCESSED_DATA_KNOWLEDGE_PATH}")
+data_processing_op(max_seq_len={MAX_SEQ_LEN}, max_batch_len={MAX_BATCH_LEN}, sdg_path="{DATA_PVC_SDG_PATH}", model_path="{DATA_PVC_MODEL_PATH}", skills_path="{PREPROCESSED_DATA_SKILLS_PATH}", knowledge_path="{PREPROCESSED_DATA_KNOWLEDGE_PATH}")
```

Comment:
Why is this function the only one receiving an update? I believe `run_final_eval_op` needs to change too. Did I miss something?

Did you update `standalone/standalone.py` manually?

Reply:
I have no idea why is this line in `standalone.tpl` and not sourced like the rest from `pipeline.py`:
https://github.com/redhat-et/ilab-on-ocp/pull/155/files#diff-6b92e3e77956f8020482ca91889a24f3411c9a4854f2aaaac30ee504ef26ede4L517-L522

I did not update `standalone.py` manually and I don't want to. 😄

Reply:
![image](https://github.com/user-attachments/assets/420edcc8-7cef-484c-b784-8ea6214a2174)